src/Bicep.Decompiler.IntegrationTests/Files/Working/issue12197/main.bicep

```diff
@@ -0,0 +1,121 @@
+@description('Location for all resources.')
+param location string
+
+@description('Base name that will appear for all resources.')
+param baseName string = 'adecosmosapp2'
+
+@description('Three letter environment abreviation to denote environment that will appear in all resource names')
+param environmentName string = 'cicd'
+
+@description('App Service Plan Sku')
+param appServicePlanSKU string = 'D1'
+
+@description('Resource Group Log Analytics Workspace is in')
+param logAnalyticsResourceGroup string
+
+@description('Log Analytics Workspace Name')
+param logAnalyticsWorkspace string
+
+@description('Resource Group CosmosDB is in')
+param cosmosDBResourceGroup string
+
+@description('CosmosDB Name')
+param cosmosDBName string
+
+@description('Dev Center Project Name')
+param devCenterProjectName string = ''
+
+@description('Name for the Azure Deployment Environment')
+param adeName string = ''
+
+var regionReference = {
+  centralus: 'cus'
+  eastus: 'eus'
+  westus: 'wus'
+  westus2: 'wus2'
+}
+var language = 'Bicep'
+var nameSuffix = (empty(adeName) ? toLower('${baseName}-${environmentName}-${regionReference[location]}') : '${devCenterProjectName}-${adeName}')
+
+module userAssignedIdentityModule './nested_userAssignedIdentityModule.bicep' = {
+  name: 'userAssignedIdentityModule'
+  params: {
+    location: location
+    userIdentityName: nameSuffix
+  }
+}
+
+module appServicePlanModule './nested_appServicePlanModule.bicep' = {
+  name: 'appServicePlanModule'
+  params: {
+    location: location
+    appServicePlanName: nameSuffix
+    language: language
+    appServicePlanSKU: appServicePlanSKU
+    appServiceKind: 'linux'
+  }
+}
+
+module appServiceModule './nested_appServiceModule.bicep' = {
+  name: 'appServiceModule'
+  params: {
+    location: location
+    appServicePlanID: reference(appServicePlanModule.id, '2022-09-01').outputs.appServicePlanID.value
+//@[53:55) [BCP053 (Error)] The type "module" does not contain property "id". Available properties include "name", "outputs". (CodeDescription: none) |id|
```

Comment:
Maybe not related to the changes in this PR, but shouldn't this have decompiled to `appServicePlanModule.outputs.appServicePlanID`?

Reply:
Yeah seems like we're not handling module outputs correctly at all.

Reply:
@jeskew could you create a new issue?

Reply:
And can you send it to me when you create it. I might just pick that up since I already looked around in the decompiler code for this issue.